bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/editor/Institutions.scala

```diff
@@ -74,6 +81,6 @@ object Contact {
 case class Contact(phone: List[String], email: List[String])
 
 object Institution {
-  val empty = Institution("", Nil, "", Contact.empty)
+  val empty = Institution("", Nil, "", None, Contact.empty)
 }
-case class Institution(name: String, addr: List[String], country: String, contact: Contact)
+case class Institution(name: String, addr: List[String], country: String, affiliate: Option[String], contact: Contact)
```

Comment:
Could/Should affiliate be `Option[NgoPartner (or the correct type for an affiliate)]`?